forch/device_report_client.py

```diff
@@ -142,3 +142,9 @@ def process_port_assign(self, mac, vlan):
         with self._lock:
             self._mac_assigned_vlan_map[mac] = vlan
             self._process_session_ready(mac)
+
```

Comment:
How is this conceptually different than the previous function?   When would they be used differently?  If they're more the same than not, I would advocate for changing the name of the existing function to be more appropriate to the generalized use rather than creating a duplicate function "just because it's used from a different place"...